### ISSUE TITLE
MPEG1 incorrectly parsed

### ISSUE THREAD

**Issue by Sami32** [UniversalMediaServer/UniversalMediaServer#900]
Hello,

It will be good to know if others people get also this problem with other OS and renderer.
MediaInfo standalone result : [MPEG1.txt](https://github.com/UniversalMediaServer/UniversalMediaServer/files/284546/MPEG1.txt)

`MediaInfo = true`

When i try to stream to my _Panasonic Viera_ renderer an **MPEG1** file, as correctly identified by MediaInfo, **UMS recognize it wrongly** :

> TRACE 2016-05-26 13:25:29.359 [New I/O worker #2] Matched support line f:mpegps        v:mpeg2                   a:ac3|lpcm|mpa                                     m:video/mpeg
> TRACE 2016-05-26 13:25:29.359 [New I/O worker #2] File "data0001.mpg" will be sent with MIME type "video/mpeg"
> TRACE 2016-05-26 13:25:29.359 [New I/O worker #2] Adding "data0001.mpg" to transcode folder for player: "null"

It matched a line where MPEG1 was not mentioned !
The others type of files are correctly recognized.
[UMS-debug_TranscodeAudio=WAV&data0001.mpg.log.txt](https://github.com/UniversalMediaServer/UniversalMediaServer/files/284550/UMS-debug_TranscodeAudio.WAV.data0001.mpg.log.txt)

Is anything to do with _LibMediaInfoParser.java_ ?
```
		} else if (value.startsWith("version 1")) {
			if (media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.MPEG2) && audio.getCodecA() == null) {
				format = FormatConfiguration.MPEG1;
```

**Comment by valib** [comment]
@Sami32 thanks for reporting. Fixed in 5f7bad5

**Comment by Sami32** [comment]
@valib Thank you for the fix, i will try it this week-end :-)
By the way, i tried `gmc:0|3` for AVI files and it was working fine, so maybe it could work for you as well ?
